modules/pam_mkhomedir/mkhomedir_helper.c

```diff
@@ -319,9 +320,16 @@ create_homedir(struct dir_spec *parent, const struct passwd *pwd,
    retval = PAM_SUCCESS;
 
  go_out:
+
```

Comment:
Why this extra blank line?

Reply:
Thanks for the review. 
Unwanted blank line removed with the last change